driver/pace/driver/tools.py

```diff
@@ -27,14 +27,48 @@
     type=click.STRING,
 )
 @click.option("--report_detail", is_flag=True, type=click.BOOL, default=False)
-def command_line(action: str, sdfg_path: Optional[str], report_detail: Optional[bool]):
+@click.option(
+    "--hardware_bw_in_gb_s",
+    required=False,
+    type=click.FLOAT,
+    default=0.0,
```

Comment:
Is the idea that if you pass `None` here you calculate the bandwidth?

Reply:
Yes. Technically you should always run something local to test your own hardware. But the stencil I am using is not necessarily the best, and you might want to run specifics analysis with hardware you are not running on.